lib/SS2K/include/Constants.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2020  Anthony Doud & Joel Baranick
+ * All rights reserved
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+#pragma once
+
+#include <NimBLEUUID.h>
+
+#define CYCLINGPOWERMEASUREMENT_UUID NimBLEUUID((uint16_t)0x2A63)
```

Comment:
I'm perfectly okay with moving the UUID's out of BLE common.h but not okay with splitting them up. They all need to be moved here or left as a group over there.

Reply:
Okay. I was looking at this as an interim step. I’m fine moving them all here now. I think much of the advertising code moves to the lib, so it can be easily tested.